monkey/tests/resources/server_config_with_credentials.json

```diff
@@ -0,0 +1,4 @@
+{
+  "server_config": "password",
+  "deployment": "develop"
+}
```

Comment:
No credentials? Also, where is this being used? Is this different from [`monkey/tests/resources/environment/server_config_with_credentials.json`](https://github.com/guardicore/monkey/blob/e3a0105b04795ed460de42450c1b64085781bd3e/monkey/tests/resources/environment/server_config_with_credentials.json)?

Reply:
Looks like those got added accidentally. I'll remove them.